src/remediations/write.integration.js

```diff
@@ -234,6 +234,112 @@ describe('remediations', function () {
                 title: 'Unsupported Content-Type application/xml (location: undefined, path: undefined)'
             }]);
         });
+
+        test('500s when remediation is created with name that is used by existing remediation created by the same user', async () => {
+            const name = 'duplicate name and same user for create';
+            const {id, header} = reqId();
+
+            const r1 = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.testWrite)
+            .send({name})
+            .expect(201);
+
+            const {body} = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.testWrite)
+            .send({name})
+            .expect(500);
```

Comment:
500 is an internal server error.  I know the remediations code is a bit fragile but we should probably be returning a 400 (with an appropriate message) for duplicate remediation plan name, right?  (500 would be a bug...)

Reply:
Yeah, I was wondering about this. Maybe the constraint isn't being setup correctly because a 500 error(not 400) is being returned when you try to create/update a remediation with a duplicate name. I'm gonna look into the constraint more.